app/views/users/confirmation.html.erb

```diff
@@ -1,20 +1,20 @@
 <!--Descriptive page name, messages and instructions-->
-<h2 class="heading"><%= ts("Almost Done!") %></h2>
+<h2 class="heading"><%= t(".page_heading") %></h2>
 <!--/descriptions-->
 <!--main content-->
 <div class="userstuff">
   <p>
-    <%= ts("You should soon receive a confirmation email at the address you gave us. This will have the link you need to follow in order to activate your account and complete the account creation process. The confirmation email will come from <strong>%{return_address}</strong> -- you might want to add this to your address book to make sure you get the email.", return_address: ArchiveConfig.RETURN_ADDRESS).html_safe %>
+    <%= t(".receive_email_html", return_address: ArchiveConfig.RETURN_ADDRESS) %>
   </p>
   <p>
-    <%= ts("If you haven't received this email within 24 hours, and you don't find our email in your spam filter or Social folder, please <a href='%{support_path}'>contact Support</a> for help.", support_path: new_feedback_report_path).html_safe %>
+    <%= t(".no_email_html", contact_support_link: (link_to t(".contact_support"), new_feedback_report_path)) %>
   </p>
   <p>
     <% # We do days_to_purge_unactivated * 7 because it's actually weeks %>
```

Comment:
**(Optional)** Oh -- while you're here, could you change this to use the the [ERB-specific comment syntax](https://ruby-doc.org/3.2.2/stdlibs/erb/ERB.html#class-ERB-label-Recognized+Tags)? From what I've seen, Rails is more lenient than the link suggests, but it should make syntax highlighting on GitHub more reasonable (and maybe even help Hound detect things like the quotes below)
```suggestion
    <%# We do days_to_purge_unactivated * 7 because it's actually weeks %>
```